packages/api-clients/open-api/delegationApi.yml

```diff
@@ -0,0 +1,461 @@
+      - name: delegationId
+        in: path
+        description: The delegation id
+        required: true
+        schema:
+          type: string
+    get:
+      description: Retrieves a delegation
+      summary: Retrieves a delegation
+      tags:
+        - producer
+      operationId: getProducerDelegation
+      responses:
+        "200":
+          description: Delegation retrieved
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Delegation"
+        "401":
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "404":
+          description: Delegation not found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+    delete:
+      description: Revokes a delegation
+      tags:
+        - producer
+      summary: Revoke a delegation
+      operationId: revokeProducerDelegation
+      responses:
+        "204":
+          description: Delegation revoked
+        "401":
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "403":
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+        "404":
+          description: Delegation not found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem"
+components:
+  schemas:
+    CompactDelegation:
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+          format: uuid
+        eserviceName:
+          type: string
+        delegatedName:
+          type: string
+        delegatorName:
+          type: string
+        state:
+          $ref: "#/components/schemas/DelegationState"
+      required:
+        - id
+        - eserviceName
+        - delegatedName
+        - delegatorName
+        - state
+    CompactDelegations:
+      type: object
+      additionalProperties: false
+      properties:
+        results:
+          type: array
+          items:
+            $ref: "#/components/schemas/CompactDelegation"
+        totalCount:
+          type: integer
+          format: int32
+      required:
+        - results
+        - totalCount
+    CreatedResource:
+      type: object
+      additionalProperties: false
+      description: contains the id of the created resource
+      properties:
+        id:
+          type: string
+          format: uuid
+      required:
+        - id
```

Comment:
The listing in the process should return the array of `Delegation` component, `CompactDelegation` should be removed